src/components/TextField/TextField.vue

```diff
@@ -237,7 +253,7 @@ interface NonMutuallyExclusiveProps {
   /** Limit increment value for numeric and date-time inputs */
   step?: number;
   /** Enable automatic completion by the browser. Set to "off" when you do not want the browser to fill in info */
-  autoComplete: string;
+  autoComplete?: string;
```

Comment:
if this field is not required, so it should have default value, because Shopify requires this for accessibility.